packages/manager/apps/web/client/app/hosting/dashboard/hosting.routes.js

```diff
@@ -99,7 +103,7 @@ export default /* @ngInject */ ($stateProvider) => {
           serviceName,
         ).catch(() => []),
       emailOptionIds: /* @ngInject */ (hostingEmailService, serviceName) =>
-        hostingEmailService.getEmailOptionList(serviceName),
+        hostingEmailService.getEmailOptionList(serviceName).catch(() => null),
```

Comment:
return null if service is expired otherwise throw the error so error page is displayed.

Reply:
We are catching `406 service expired error` for the main API, hence this will not need a catch block. As we have lot of APIs other than main API, we should not be handling subsequent API errors.